.github/workflows/Dockerfile.dev.yml

```
@@ -21,6 +21,8 @@ concurrency:
 env:
   HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
   IMAGE: ${{ format('{0}/{1}:{2}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.repository, github.event_name == 'pull_request' && format('pr-{0}-dev', github.event.pull_request.number) || 'dev') }}
+  GO_REPO_BRANCH=soroban-xdr-next
+  CORE_REPO_BRANCH=master
```

This is YAML, not bash 😊
```suggestion
  GO_REPO_BRANCH: soroban-xdr-next
  CORE_REPO_BRANCH: master
```